docs/conf.py

```diff
@@ -58,9 +59,30 @@
 # -- Options for HTML output -------------------------------------------------
 html_theme = "nvidia_sphinx_theme"
 html_static_path = ["_static"]
+html_extra_path = ["project.json", "versions1.json"]
```

Comment:
Silly question out of curiosity, non-blocking: why the `1` in `versions1.json`, instead of `versions.json`?